src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs

```diff
@@ -143,6 +143,27 @@ public void LogDependency_SinksToApplicationInsights_ResultsInDependencyTelemetr
             }
         }
 
+        [Fact]
+        public void LogServiceBusDependency_SinksToApplicationInsights_ResultsInServiceBusDependencyTelemetry()
+        {
+            // Arrange
+            using (ILoggerFactory loggerFactory = CreateLoggerFactory())
+            {
+                ILogger logger = loggerFactory.CreateLogger<ApplicationInsightsSinkTests>();
+
+                string entityName = _bogusGenerator.Commerce.Product();
+                bool isSuccessful = _bogusGenerator.PickRandom(true, false);
+                TimeSpan duration = _bogusGenerator.Date.Timespan();
+                Dictionary<string, object> telemetryContext = CreateTestTelemetryContext();
+
+                // Act
+                logger.LogServiceBusDependency(entityName, isSuccessful, DateTimeOffset.UtcNow, duration, ServiceBusEntityType.Queue, telemetryContext);
```

Comment:
Can we align the timestamp with other tests?

Reply:
Yep, done that.